package.json

```diff
@@ -13,9 +13,7 @@
   "typings": "lib/index.d.ts",
   "files": [
     "bin/",
-    "lib/",
-    ".scripts/",
-    "cached_node_modules/"
+    "lib/"
```

Comment:
it is in `lib/`. I move the current node_modules temporarily to `.tmp/` and install a fresh bundle. After publishing the initial node_modules is moved back to the root